lib/std/zig/system.zig

```diff
@@ -1020,8 +1020,8 @@ fn resolveElfFileRecursively(cwd: fs.Dir, start_path: []const u8) error{UnableTo
     var buffer: [255 + 1]u8 = undefined;
     while (true) {
         // Interpreter path can be relative on Linux, but
-        // for simplicity we are asserting it is an absolute path.
-        assert(std.fs.path.isAbsolute(current_path));
+        // for simplicity we are assuming it is an absolute path.
+        // It can also be a relative path from PATH environment variable.
```

Comment:
"we are assuming it is an absolute path"
"it can be a relative path"

these are contradictory statements. Can you either make the comment accurate or delete it?

Reply:
I wanted to mean here "assume interpreter path is absolute, and handle both absolute and relative for PATH", if it's not clear I can rewrite it

Reply:
It's pretty confusing, but, it's OK. Better to fix the regression now.